vector/src/main/res/layout/coach_mark.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:id="@+id/coach_mark"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="start|top"
+        android:paddingHorizontal="20dp"
+        android:orientation="vertical"
+        tools:ignore="UselessParent">
+
+        <ImageView
+            android:id="@+id/coach_mark_arrow_top"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="-1dp"
+            android:scaleType="fitXY"
+            android:src="@drawable/ic_coach_arrow"
+            app:tint="?colorPrimary"
+            tools:ignore="ContentDescription"/>
+
+        <TextView
+            android:id="@+id/coach_mark_content"
```

Comment:
You should use `style`, depending of the text size (16sp here).
Here we will have: `style="@style/Widget.Vector.TextView.Subtitle"`.

`android:textAppearance` and `android:textSize` attributes can be removed and should never be used directly (project convention).